app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryList2Fragment.kt

```diff
@@ -54,6 +54,12 @@ class ContentEntryList2Fragment : UstadListViewFragment<ContentEntry, ContentEnt
             field = value
         }
 
+    override var hasUpdatePermission: Boolean = false
```

Comment:
As this is the view let's give it a view like property name (the view doesn't care about underlying permissions etc) e.g. editOptionVisible